CHANGELOG.rst

```diff
@@ -1,6 +1,37 @@
 Changelog
 =========
 
+1.1.0 (2021-04-26)
+------------------
+
+- [NEW] Implemented the ``dehumanize`` method for ``Arrow`` objects. This takes human readable input and uses it to perform relative time shifts, for example:
+
+.. code-block:: python
+
+    >>> arw
+    <Arrow [2021-04-26T21:06:14.256803+00:00]>
+    >>> arw.dehumanize("8 hours ago")
+    <Arrow [2021-04-26T13:06:14.256803+00:00]>
```

Comment:
Since this is a big new feature, should we include a couple more examples?

Reply:
Sure any particular ones you'd like to include?

Reply:
Maybe three more:
1. English string with 3 timeframes
2. Future english string
2. Another locale